controller/relay-image.go

```diff
@@ -83,14 +86,18 @@ func relayImageHelper(c *gin.Context, relayMode int) *OpenAIErrorWithStatusCode
 	userQuota, err := model.CacheGetUserQuota(userId)
 
 	sizeRatio := 1.0
-	// Size
-	if imageRequest.Size == "256x256" {
-		sizeRatio = 1
-	} else if imageRequest.Size == "512x512" {
-		sizeRatio = 1.125
-	} else if imageRequest.Size == "1024x1024" {
-		sizeRatio = 1.25
+
+	if ratio, ok := common.DalleSizeRatios[imageModel][requestSize]; ok {
```

Comment:
if ratio not found in map `DalleSizeRatios`, there should be a invalid `requestSize` field error